### PR TITLE
Update data-loss-prevention-policies.md

### PR DESCRIPTION
**Pull request by FBrownMSFT**
doc update

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@FBrownMSFT - PR #610  appears to be a duplicate to this PR. Which of the two is the correct one?

Please confirm and close the PR that will not be used. Thank you.

File: microsoft-365/compliance/data-loss-prevention-policies.md [hunk 1]
```diff
@@ -333,8 +333,10 @@ By using labels as a condition in your DLP rules, can you selectively enforce pr
 > [!NOTE]
 > If you specify a retention label as a condition in a DLP policy and you also include Exchange and/or Teams as a location, you will receive the following error:
 
-~~~powershell"Protecting labeled content in email and teams messages isn't supported. Either remove the label below or turn off Exchange and Teams as a location."
-```
+~~~powershell
+"Protecting labeled content in email and teams messages isn't supported. Either remove the label below or turn off Exchange and Teams as a location."
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-loss-prevention-policies.md]:
```suggestion
"Protecting labeled content in email and Teams messages isn't supported. Either remove the label below or turn off Exchange and Teams as a location."
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by markjjo** [comment]
Closing as this fix was made in another PR

**Pull request closed without merging**